variables.json

```diff
@@ -161,6 +161,105 @@
     "xsl_filename": "transforms/qdcCDMingest.xsl",
     "xsl_repository": "tulibraries/aggregator_mdx"
   },
+  "WESTCHESTER_OAI_CONFIG": {
+    "endpoint": "http://digital.klnpa.org/oai/oai.php",
+    "md_prefix": "oai_qdc",
+    "all_sets": "False",
+    "excluded_sets": [],
+    "included_sets": [
+     "p17189coll1",
+      "p17189coll4",
+      "chester",
+      "diaries",
+      "philips",
+      "qwcaralia",
+      "qwcarch",
+      "qwccivilwar",
+      "qwcphoto",
+      "qwcpost",
+      "redware",
+      "sharples",
+      "wcnp01",
+      "wcudoll",
+      "wcutreasure",
+      "wpa"
+    ],
+    "schematron_filter": "validations/dcingest_reqd_fields.sch",
```

Comment:
should be qdcingest_reqd_fields.sch in all OAI configs

Reply:
ALL of them or just the KLNs?